app/setup.py

```diff
@@ -326,7 +340,9 @@ def setup_submitter(application):
         bucket_name = application.config.get("EQ_GCS_SUBMISSION_BUCKET_ID")
 
         if not bucket_name:
```

Comment:
Could be simplified with a conditional e.g `if bucket_name :=` and then `else`. so we can remove `if not`

Reply:
UPDATE: changed back to old way suggested by Mebin, but implemented walrus operator 👍